src/page-objects/storefront/AccountLogin.ts

```diff
@@ -35,6 +36,7 @@ export class AccountLogin implements PageObject {
         this.streetAddressInput = this.billingAddressFormArea.getByLabel('Street address*');
         this.cityInput = this.billingAddressFormArea.getByLabel('City*');
         this.countryInput = this.billingAddressFormArea.getByLabel('Country*');
+        this.postalCodeInput = this.billingAddressFormArea.getByLabel('Postal code');
```

Comment:
I guess now postal code is a mandatory area, but why don't we add "*" to this snippet?

Reply:
It depends on the country to be mandatory or not, since not sure I just getByLabel('Postal code') instead of getByLabel('Postal code*')

Reply:
But if it is mandatory in a specific country, does it get an "*" at the end?

Reply:
@yusufttur but i have not tested it to see if it works correctly and i do not know how to test it

Reply:
But if it is mandatory in a specific country, does it get an "*" at the end? yes

Reply:
Yes, please update and add "*".